package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-vis",
-  "version": "0.6.5",
+  "version": "0.6.6",
```

Comment:
We try to keep version bumps outside of PRs. That way, we can keep commit history readable/clean, and the version bump/publishing process tied closely together.